src/patternfly/components/NotificationBadge/notification-badge.scss

```diff
@@ -39,17 +67,39 @@
     transform: translate(var(--pf-c-notification-badge--after--TranslateX), var(--pf-c-notification-badge--after--TranslateY));
   }
 
+  .pf-c-button & {
+    position: static;
```

Comment:
is there a use of the notification badge outside of a button? It seems like the negative left/right margin assumes this will always be placed inside of a button?

Reply:
I don't think so. I removed the negative margin as this presents the badge inconsistently with the other buttons.

Reply:
could `position: static` just apply to `.pf-c-notification-badge`?

Reply:
We could if the notification badge will only ever be used inside a button

Reply:
Yeah that's what I meant with my original question. Is there a use case for it outside of a button? Seems like for this layout to work, it needs to be `static`, and placed in a non-`static` parent for the pseudo elements to work correctly.